src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
`ValidateSet` is reasonable, especially if there is an easy way to test.

I would like to see this command converted to C#.